src/sections/Gsod/gsod.style.js

```diff
@@ -0,0 +1,123 @@
+import styled from "styled-components";
+
+
+const GSODSectionWrapper = styled.div`
+element.style {
+    font-size: 0.7em;
+    padding-bottom: 40px;
+    padding-top: 10px;
+    width: 300px;
+}
+.btn {
+    background-color: #3eb5a1;
+}
+.waves-effect {
+    position: relative;
+    cursor: pointer;
+    display: inline-block;
+    overflow: hidden;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+    -webkit-tap-highlight-color: transparent;
+    vertical-align: middle;
+    z-index: 1;
+    -webkit-transition: 0.3s ease-out;
+    transition: 0.3s ease-out;
+}
+.btn, .btn-large {
```

Comment:
@Tanuj22 what is our strategy with respect to the reuse of global styles?

Reply:
These styles won't be needed. We are going to use a separate button component already made. (Was just waiting for @codetheorem to app a temp link for doing a review)